lib/android/bundle/bundle.nit

```diff
@@ -63,7 +63,7 @@ extern class NativeBundle in "Java" `{ android.os.Bundle `}
 	`}
 	# FIXME: Java's `char` are encoded on 16-bits whereas Nit's are on 8-bits.
```

Comment:
Unfortunately, no, it is not fixed, it will be once we have UTF-8 properly supported, until then, the cast as pictured here will be sufficient